modules/sdpolicy-security.adoc

```diff
@@ -77,7 +77,8 @@ $ oc adm policy add-cluster-role-to-group self-provisioner system:authenticated:
 
 [id="network-security_{context}"]
 == Network security
-With {product-title} on AWS, AWS provides a standard DDoS protection on all Load Balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing Load Balancers used for {product-title}. A 10-second timeout is added for HTTP requests coming to the haproxy router to receive a response or the connection is closed to provide additional protection.
+Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules (AWS Security Groups or Google Cloud Compute Engine firewall rules). {product-title} customers on AWS are also protected against DDoS attacks with link:https://docs.aws.amazon.com/waf/latest/developerguide/ddos-overview.html[AWS Shield Standard].
```

Comment:
Maybe rephrase to avoid parenthesis. 
```suggestion
Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules, either AWS Security Groups or Google Cloud Compute Engine firewall rules.
```